core/src/main/scala/akka/kafka/javadsl/Producer.scala

```diff
@@ -152,6 +153,116 @@ object Producer {
       producer: org.apache.kafka.clients.producer.Producer[K, V]
   ): Sink[Envelope[K, V, ConsumerMessage.Committable], CompletionStage[Done]] = committableSink(settings, producer)
 
+  /**
+   * Create a sink that is aware of the [[ConsumerMessage.CommittableOffset committable offset]]
```

Comment:
Typo `[[ConsumerMessage.CommittableOffset committable offset]]`.  The other new factory methods in the Scala and Java DSLs have the same typo.

Reply:
It's not a typo a link to a class or method can use any text, but I figured these links should link to `Committable` instead.

Reply:
Got it.  I misunderstood the scaladocs syntax.